### ISSUE TITLE
Coverage not written when using `ember serve`

### ISSUE THREAD

**Issue by avdv** [sglanzer/ember-cli-blanket#68]
When using `ember serve` and running the tests in the browser, I receive an error in the console (I'm proxying requests to an upstream server):

```
POST http://localhost:4200/write-blanket-coverage 404 (Not Found)
```

Apparently, the coverage middleware does not get started in this mode, only when it is run via testem, ie. when running `ember test`.

Does it make sense to make ember-cli-blanket a serverMiddleware so that is also started with `ember serve`?

**Comment by jschilli** [comment]
@avdv I removed coverage for serverMiddleware in https://github.com/sglanzer/ember-cli-blanket/commit/9b63dfd3dbf6a469fca3cf810630536d7062ee71

that was an expedient fix to #53  and #46 

I'm open to bringing it back as I agree with having the ability to write it from `ember serve` as well.

interested in PR'ing that?

**Comment by avdv** [comment]
> interested in PR'ing that?

Sure, seems easy enough: reverting 9b63dfd will probably do it.
